CONTRIBUTORS.yaml

```diff
@@ -2710,3 +2719,11 @@ tbrown91:
     email: brown@izw-berlin.de
     orcid: 0000-0001-8293-4816
     joined: 2024-08
+
+rmassei:
+    name: Riccardo Massei
+    email: riccardo.massei@ufz.de
+    orcid: 0000-0003-2104-9519
+    joined: 2024-11
+    affiliations:
+      - ufz
```

Comment:
I think you also need to add the UFZ

Reply:
ok, just left nfdi4bioimage, I also realized that my name was 2 times in the contributors list